### ISSUE TITLE
custom claims

### ISSUE THREAD

**Issue by sw-yx** [sw-yx/react-netlify-identity#8]
https://auth0.com/docs/scopes/current/custom-claims

**Comment by jahvi** [comment]
Not sure if this is part of an internal todo but it's the only thing I could find, are custom claims supported in Netlify Identity?

**Comment by sw-yx** [comment]
they could be supported via [user metadata](https://www.netlify.com/docs/identity/#identity-user-metadata) but i was just stubbing out this issue as an idea for adding some code examples and stuff to make this work. if you'd like to mock up something i'd be open to collaborating, but before you i hadn't really prioritized it or anything. was just an idea.

**Comment by jahvi** [comment]
I started playing around with it and user metadata works for my use case, the only thing I found in the library is that the [`updateUser`](https://github.com/sw-yx/react-netlify-identity/blob/master/src/index.tsx#L46) function doesn't expect a regular Object as documented but instead it's this format `{ data: { ... } }`.

I spent some time wondering why the user data wasn't getting updated 😄

**Comment by sw-yx** [comment]
yeah sorry even *i* havent got around to using it and i work here lol

docs PR very very welcome!!!

**Comment by jahvi** [comment]
No worries it happens 😄

I just submitted a PR #15 let me know if you want me to change anything.
